website/docs/styles/border.md

```diff
@@ -0,0 +1,208 @@
+| strokeAlign | `StacDouble?`  | Defines the relative position of the stroke on values range from -1.0 (inside) to 1.0 (outside). |
+
+### Individual Border Sides
+
+| Property | Type              | Description                             |
+| -------- | ----------------- | --------------------------------------- |
+| top      | `StacBorderSide?` | Defines the border for the top side.    |
+| right    | `StacBorderSide?` | Defines the border for the right side.  |
+| bottom   | `StacBorderSide?` | Defines the border for the bottom side. |
+| left     | `StacBorderSide?` | Defines the border for the left side.   |
+
+:::note
+
+When any individual border side is specified (`top`, `right`, `bottom`, or `left`), the system will use individual border mode and ignore the legacy uniform border properties (`color`, `width`, `borderStyle`, `strokeAlign`).
+
+:::
+
+## Examples
+
+### Uniform Border (All Sides)
+
+```json
+{
+  "border": {
+    "color": "#FF0000",
+    "width": 2.0,
+    "borderStyle": "solid",
+    "strokeAlign": 0.0
+  }
+}
+```
+
+### Individual Border Sides
+
+```json
+{
+  "border": {
+    "top": {
+      "color": "#FF0000",
+      "width": 2.0,
+      "borderStyle": "solid",
+      "strokeAlign": 0.0
+    },
+    "right": {
+      "color": "#00FF00",
+      "width": 1.0,
+      "borderStyle": "solid"
+    },
+    "bottom": {
+      "color": "#0000FF",
+      "width": 3.0,
+      "borderStyle": "solid"
+    },
+    "left": {
+      "color": "#FFFF00",
+      "width": 1.5,
+      "borderStyle": "solid"
+    }
+  }
+}
+```
+
+### Partial Border Sides
+
+You can specify only some sides, and the others will default to `BorderSide.none`:
+
+```json
+{
+  "border": {
+    "top": {
+      "color": "#FF0000",
+      "width": 2.0,
+      "borderStyle": "solid"
+    },
+    "bottom": {
+      "color": "#0000FF",
+      "width": 2.0,
+      "borderStyle": "solid"
+    }
+  }
+}
+```
+
+### Complete Container Example
+
+```json
+{
+  "type": "container",
+  "width": 200,
+  "height": 100,
+  "decoration": {
+    "color": "#F0F0F0",
+    "border": {
+      "top": {
+        "color": "#FF0000",
+        "width": 3.0,
+        "borderStyle": "solid"
+      },
+      "right": {
+        "color": "#00FF00",
+        "width": 2.0,
+        "borderStyle": "solid"
+      },
+      "bottom": {
+        "color": "#0000FF",
+        "width": 3.0,
+        "borderStyle": "solid"
+      },
+      "left": {
+        "color": "#FFFF00",
+        "width": 2.0,
+        "borderStyle": "solid"
+      }
+    },
+    "borderRadius": {
+      "topLeft": 8.0,
+      "topRight": 8.0,
+      "bottomLeft": 8.0,
+      "bottomRight": 8.0
+    }
+  },
+  "child": {
+    "type": "center",
+    "child": {
+      "type": "text",
+      "data": "Custom Borders",
+      "style": {
+        "fontSize": 16,
+        "fontWeight": "bold"
+      }
+    }
+  }
+}
+```
+
+## Migration Guide
```

Comment:
Remove the migration part please. We don't want people to migrate to individual sides if they are using uniform border.